collections/guides/manage/manage-applications-terraform.md

```diff
@@ -27,7 +27,7 @@ created an Application under your tenant using Terraform, and you will have an A
 
 <span class="base-alert warning">
   <span>
-    To start, you'll need a new <code>Management</code> [Application](https://docs.basistheory.com/api-reference/#applications) with <code>application:create</code>, <code>application:read</code>, <code>application:update</code>, and <code>application:delete</code> permissions. <a href="https://portal.basistheory.com/applications/create?name=Terraform+Application&permissions=application%3Acreate&permissions=application%3Aread&permissions=application%3Aupdate&permissions=application%3Adelete&type=management" target="_blank">Click here to create one.</a>
+    To start, you'll need a new <code>Management</code> [Application](https://docs.basistheory.com/api-reference/#applications) with <code>application:create</code>, <code>application:read</code>, <code>application:update</code>, and <code>application:delete</code> permissions. <a href="https://portal.basistheory.com/applications/create?application_template_id=931b306f-6be7-405d-9a8d-1442a99dd2d7" target="_blank">Click here to create one.</a>
```

Comment:
not sure if this template has more complex rules than just simple permissions. if it has more permissions than we say here, we still have the option to keep the query params how they were since these are all simple permissions for management. this comment goes for all of the management ones

Reply:
i will check all of them again to make sure